publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -59,6 +63,15 @@ public class RabbitMqProperties {
 
     Logger log = (Logger) LoggerFactory.getLogger(RMQHelper.class);
 
+
+    public Long getWaitForConfirmsTimeOut() {
+        return WaitForConfirmstimeOut;
```

Comment:
- WaitForConfirmstimeOut should start with lower-case letter, as an instance variable.
- WaitForConfirmstimeOut -> waitForConfirmsTimeOut